x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -494,6 +494,11 @@ export interface HostInfoInterface {
      */
     endpoint: PolicyInfo;
   };
+  /**
+   * The time when the Elastic Agent associated with this Endpoint host checked in with fleet
+   * Same as Agent['last_checkin']
+   */
+  last_checkin?: UnitedAgentMetadataPersistedData['united']['agent']['last_checkin'];
```

Comment:
Ohhh. I just noticed this and perhaps that's why in the UI you are using conditional logic on the date.

Why is it condition? It should not be (IMO) - even if the United index type has it as condition. If thats the case, then just have the API default back to the prior date.

Reply:
done0575e8e5a5384a2bc59a7dd2553752a407fc5fc3